media/js/base/core-datalayer.js

```diff
@@ -25,6 +25,33 @@ if (typeof Mozilla == 'undefined') {
         return pageId ? pageId : pathName.replace(/\/[^\/]*/, '');
     };
 
+    /** Returns whether page has download button.
+    * @param {String} path - URL path name fallback if page ID does not exist.
+    * @return {String} boolean.
+    */
+    Analytics.pageHasDownload = function() {
+        return ( $('[data-download-os]').length > 0 ) ? 'true' : 'false';
```

Comment:
Good call on the shortening.  And I went with using the data attribute over the class `.download-link` becasue I was not sure if all download link had that class and but I know that all download links do have `data-download-os`, or they should at least.